string robot_name
Point3d[] path
